checkstyle.xml

```diff
@@ -202,7 +202,7 @@
             <property name="severity" value="ignore"/>
         </module>
 	    <module name="MethodLength">
-            <property name="max" value="200"/>
```

Comment:
Don't change the global limit. 300 is double the default 
http://checkstyle.sourceforge.net/config_sizes.html#MethodLength

In most cases, you shouldn't need to, but if you have a specific method that needs to be longer, you can disable the check for that specific method via either comment or annotation. http://checkstyle.sourceforge.net/config_annotation.html#SuppressWarnings